src/app/templates/_bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "<%= nameOf.package %>",
+  "version": "0.0.0",
```

Comment:
The issue is that then you end up with a lot of plugins are basically permanently in the 0.x zone. Which is why npm chose to change the default.